components/ParameterForm.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-  <div>
+  <div v-show="pageMounted">
```

Comment:
is this for playwright again? i feel its causing another render now... probs best to take the L in the playwright test rather than decrease app performance and increase bloat

Reply:
Another render?

Reply:
Let me know if you manage to reproduce the re-rendering because I can't make it happen for me -- would be great to have steps to reproduce.

Reply:
nah you are good i just thought it would